### DISCUSSION TITLE
Help for compile - 4.2.7 - Linear Advance

### DISCUSSION THREAD

**Discussion by Binroo** [Jyers/Marlin#1396] [Q&A] [answered]
Please
Anyone have a procedure or tutorial for building their own version? Which file to take and how to modify the motherboard ?
I would like to switch to a 4.2.7 to activate linear advance on BLtouch 5x5

**Comment by msdossajo** [comment, marked as answer]
@Binroo you don't need to switch to a 4.2.7 to use linear advance.
Check discussions #707
For the building your own version you can get some examples from [https://github.com/Jyers/Marlin/tree/Ender_3_V2_JyersUI/Configuration%20Files](url)
and in Youtube you can find lots of information on how to install/configure Visual Studio Code and compile Marlin.
